1-exercises/C-highlight-words/script.js

```diff
@@ -1,10 +1,38 @@
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+  let content = document.querySelector('#content');
+
+  // Paragraph rendering
+  let paragraphEl = document.createElement('p');
+  content.appendChild(paragraphEl);
+
+  // Dropdown
+  let selectEl = document.createElement('select');
+  content.appendChild(selectEl);
+  colours.forEach((element) => {
+    let optionEl = document.createElement('option');
+    selectEl.appendChild(optionEl);
+    optionEl.innerHTML = element;
+  });
+
+  let split = paragraph.split(' ');
+  split.forEach((element) => {
+    let spanEl = document.createElement('span');
+    spanEl.innerText = element + ' ';
+    paragraphEl.appendChild(spanEl);
+    spanEl.addEventListener('click', () => {
+      let selectValue = document.querySelector('select').value;
+      if (selectValue === 'none') {
+        spanEl.style.backgroundColor = '';
+      } else {
+        spanEl.style.backgroundColor = selectValue;
+      }
```

Comment:
Also a ternary is possible here, 

```suggestion
      const isNone = selectValue === 'none' 
      spanEl.style.backgroundColor = isNone ? '' : selectValue
```